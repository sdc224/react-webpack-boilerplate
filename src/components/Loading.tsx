import React from "react";
import LinearProgress from "@material-ui/core/LinearProgress";

const Loading: React.FC = () => {
	return <LinearProgress color="secondary" />;
};

export default Loading;
