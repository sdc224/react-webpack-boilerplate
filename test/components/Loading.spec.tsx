import React from "react";
import { render, RenderResult } from "@testing-library/react";
import Loading from "@components/Loading";

let documentBody: RenderResult;

describe("<Loading />", () => {
	beforeEach(() => {
		documentBody = render(<Loading />);
	});
	it("matches snapshot", () => {
		const { baseElement } = documentBody;
		expect(baseElement).toMatchSnapshot();
	});
});
