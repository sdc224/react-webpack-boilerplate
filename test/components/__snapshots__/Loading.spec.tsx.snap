// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> matches snapshot 1`] = `
<body>
  <div>
    <div
      class="MuiLinearProgress-root MuiLinearProgress-colorSecondary MuiLinearProgress-indeterminate"
      role="progressbar"
    >
      <div
        class="MuiLinearProgress-bar MuiLinearProgress-barColorSecondary MuiLinearProgress-bar1Indeterminate"
      />
      <div
        class="MuiLinearProgress-bar MuiLinearProgress-bar2Indeterminate MuiLinearProgress-barColorSecondary"
      />
    </div>
  </div>
</body>
`;
